setup.cfg

```diff
@@ -1,4 +1,4 @@
 [pytest]
 python_files = *.py
-addopts = --verbose --ignore setup.py --ignore utide/robustfit.py --doctest-modules --cov-report term-missing --cov .
```

Comment:
Also, I don't see that the type of functions in this project are very suitable for having examples/doctests, which work only with very small inputs and outputs.  Tiny examples could be used for the spectral code, but there is not much point in having them in the docs.  Everything in this package except solve() and reconstruct() is really for internal use; examples for a wide audience are not needed.